packages/@uppy/aws-s3-multipart/src/MultipartUploader.js

```diff
@@ -30,8 +27,6 @@ function ensureInt (value) {
 class MultipartUploader {
   #abortController = new AbortController()
 
-  #isPaused = false
```

Comment:
It's probably not a good idea to get rid of it before we fixed the pause/resume for individual files, it could come in handy (but also, it would be quite easy to add back, so why not)